recipes/daops/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "daops" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/daops-{{ version }}.tar.gz
+  sha256: 6a0c6d6fd5117e24c8a1b6b35025af82d5c7cc351ce2ad8bb9737acc38f24ac9
+
+build:
+  number: 0
+  skip: true  # [win or py<36]
```

Comment:
What's the reason for skipping on windows? It would drastically reduce the number of package builds if this were noarch.

Reply:
the build failed on windows ... maybe because of our github submodule.

Reply:
Build on windows fails because roocs-utils is not available for windows...

So you can:
* make this noarch again
* move the python version constrain to the run requirements: via `-python >=3.6`
* ignore the windows build failure on staged_recipes as noarch packages are anyway build on linux after the conversion to feedstock.... installing the package on windows will not work due to missing dependencies, which is totally fine.

Reply:
Ok. Thanks :)

Reply:
do you know why the conda workflow failed?
https://github.com/cehbrecht/staged-recipes/actions/runs/372972036

Reply:
Though I have closed this PR the workflow does not stop ... sends a message every 10 minutes ...